Run-10252-block-over-quota (#528)
run-ai/docs: docs/Researcher/scheduling/the-runai-scheduler.md

* If the newly required resources, together with currently used resources, end up within the overall Project's quota and the requested node pool(s) quota, then the workload is ready to be scheduled as part of the guaranteed quota.
* If the newly required resources together with currently used resources end up above the Project's quota or the requested node pool(s) quota, the workload will only be scheduled if there are 'spare' GPU resources within the same node pool but not part of this Project. There are nuances in this flow that are meant to ensure that a Project does not end up with an over-quota made entirely of interactive workloads. For additional details see below.

> ### Blocking Over Quota

Author: ```suggestion
### Limit Quota Over or Under Subscription
```

Run:ai provides the ability to block over subscription of quotas by *Projects* or *Departments*.

> Over quota will be blocked under the following circumstances:

Author: OK, I understand the project/department won't get updated, but the sentence here needs a clarification as to why the update is not happening.
So maybe something like:

```suggestion
You project or department will not be updated due to quota over/under subscription issues caused by:
```

> Over quota will be blocked under the following circumstances:

Reviewer: yes this sounds good

> Over quota will be blocked under the following circumstances:

Author: ```suggestion
Over quota will be limited under the following circumstances:
```

> * If a project’s quota request (with or without nodepools) increases the sum of all the projects quotas to more the department quota (for GPU and CPU, compute and memory).

Reviewer: ```suggestion
* If a project’s quota request (with or without nodepools) increases the sum of all the department projects quotas to more than the department quota (for GPU and CPU, compute and memory).
```

* If a department’s quota decrease request (with or without nodepools) causes the sum of all projects quotas to surpass the department quota (for GPU and CPU, compute and memory).

> * If a project’s quota decrease request (with or without nodepools) causes the sum of all non-preemptible workloads to surpass that of the Project (for GPU and CPU, compute and memory).

Reviewer: this point is only relevant for clusters with version 2.15+

> * If a project’s quota decrease request (with or without nodepools) causes the sum of all non-preemptible workloads to surpass that of the Project (for GPU and CPU, compute and memory).

Reviewer: ```suggestion
* If a project’s quota decrease request (with or without nodepools) causes the sum of all allocated non-preemptible workloads in that project to surpass it (for GPU and CPU, compute and memory).
```

> To configure over or under quota subscriptions:

Author: ```suggestion
To configure limiting over or under quota subscriptions:
```

1. Press the *Tools and settings* icon, then go to *General*.
2. Enable the *Limit quota over/under subscription* toggle.

> To disable the blocking of over quota subscription, disable the toggle.

Author: ```suggestion
To disable the limiting of over quota subscription, disable the toggle.
```
